gradle.properties

```diff
@@ -0,0 +1,2 @@
+group = com.github.twitch4j
+version = 1.2.1
```

Comment:
```suggestion
version = 1.3.2
```

Reply:
We don't need to update this manually, the version property should always be set as parameter from ci. A placeholder like 0.0.0 is fine, as the default version in here will never be used.

Reply:
no nned t attaching version. as @PhilippHeuer said 0.0.0 i enough for that